### PR TITLE
[bitnami/cert-manager] add extraArgs support

### LINKED ISSUE #8569: Support for extraArgs
I am facing up to this [issue of cert-manager](https://github.com/jetstack/cert-manager/pull/819)
I can't see any extraArgs argument in the helm chart.
A lot of exter[Volume|env] and so on but noting related to the args. 

The only option is the args[] itself bur as I understand, this option override the default right?

### PR DESCRIPTION
**Pull request by aoterolorenzo**
### Description of the change

- adds support for extra arguments

### Applicable issues

  - fixes #8569

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/cert-manager/values.yaml [hunk 1]
```diff
@@ -235,7 +235,14 @@ controller:
...
     type: RollingUpdate
     rollingUpdate: {}
-  ## @param controller.extraEnvVars Add extra environment variables to the Controller container
+  ## @param controller.extraArgs Extra arguments to pass to the Webhook container
+  ## E.g:
+  ## extraArgs:
+  ##   - name: FOO
+  ##     value: "bar"
+  ##
+  extraArgs: [ ]
```
Comment (javsalgar) [inline bitnami/cert-manager/values.yaml:244]: Could you remove the space between the brackets so it looks like `[]`

File: bitnami/cert-manager/values.yaml [hunk 2]
```diff
@@ -235,7 +235,14 @@ controller:
   updateStrategy:
     type: RollingUpdate
     rollingUpdate: {}
-  ## @param controller.extraEnvVars Add extra environment variables to the Controller container
+  ## @param controller.extraArgs Extra arguments to pass to the Webhook container
```
Comment (javsalgar) [inline bitnami/cert-manager/values.yaml:238]: Inshould be Controller

File: bitnami/cert-manager/values.yaml [hunk 3]
```diff
@@ -235,7 +235,14 @@ controller:
...
+  ## E.g:
+  ## extraArgs:
+  ##   - name: FOO
+  ##     value: "bar"
+  ##
+  extraArgs: [ ]
+  ## @param controller.extraEnvVars Add extra environment variables to the Controller pod
```
Comment (javsalgar) [inline bitnami/cert-manager/values.yaml:245]: pod -> container

File: bitnami/cert-manager/values.yaml (hunk 1 above)
Comment (javsalgar) [reply bitnami/cert-manager/values.yaml:244]: Apply it to the rest

**Review by javsalgar — CHANGES_REQUESTED** [review]

File: bitnami/cert-manager/values.yaml (hunk 2 above)
Comment (aoterolorenzo) [reply bitnami/cert-manager/values.yaml]: Done in 8571300

File: bitnami/cert-manager/values.yaml (hunk 3 above)
Comment (aoterolorenzo) [reply bitnami/cert-manager/values.yaml]: Done in 8571300

File: bitnami/cert-manager/Chart.yaml [hunk 4]
```diff
@@ -26,4 +26,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-cert-manager-webhook
   - https://github.com/bitnami/bitnami-docker-cainjector
   - https://github.com/jetstack/cert-manager
-version: 0.6.2
+version: 0.7.2
```
Comment (recena) [inline bitnami/cert-manager/Chart.yaml:29]: After `0.6.2`.... `0.7.0`, no?

**Pull request merged**